genestack_client/scripts/genestack_application_manager.py

```diff
@@ -500,11 +504,28 @@ def set_applications_visibility(application, app_ids, version, level):
             continue
         sys.stdout.write('%-40s ... ' % app_id)
         sys.stdout.flush()
+        wait_application_loading(application, app_id, version)
         application.invoke('setVisibility', app_id, version, level)
         sys.stdout.write('ok\n')
         sys.stdout.flush()
 
 
+def get_application_descriptor(application, application_id, version):
+    return application.invoke('getApplicationDescriptor', application_id, version)
+
+
+def wait_application_loading(application, app_id, version, seconds=5):
+    descriptor = get_application_descriptor(application, app_id, version)
+    if not descriptor['isLoaded']:
+        sys.stdout.write('\nApplication \'%s\' with version \'%s\' is not loaded yet.'
+                         ' Waiting for loading (interrupt to abort)... ' % (app_id, version))
```

Comment:
`(interrupt to abort)` is confusing.  It will interrupt not only this upload but whole command.

How does it look for user if command interrupted?  Please add console output and shell output for this case here.